### ISSUE TITLE
Artifact not found

### ISSUE THREAD

**Issue by ChristoRibeiro** [fehmicansaglam/reactivemongo-extensions#4]
Hi,

The artifact `"net.fehmicansaglam"    %%  "reactivemongo-extensions-json" % "0.10.0.2"` is not available. I got an unresolved dependencies. Maybe the documentation is too recent?

Last point. The support of $regex (#3) looks great. Could be available soon in the 0.10.0.3?

**Comment by fehmicansaglam** [comment]
You are right. It is a mistake from last night's late coding:)

`"net.fehmicansaglam" %% "reactivemongo-extensions" % "0.10.0.2"` should work.

I am trying to finish and release 0.10.0.3 with regex and fixtures support. The only missing thing is fixtures support for bson. I think I will be able to release it before weekend.

**Comment by fehmicansaglam** [comment]
BTW it is already released with the snapshot version. This time you should use reactivemongo-extensions-json.

``` scala
resolvers += "Sonatype Snapshots" at "http://oss.sonatype.org/content/repositories/snapshots/"

libraryDependencies ++= Seq(
  "net.fehmicansaglam" %% "reactivemongo-extensions-json" % "0.10.0.3-SNAPSHOT"
)
```

**Comment by ChristoRibeiro** [comment]
Great news :)

Could be good to have a tiny changelog.md somewhere to see evolutions between release. It's just an idea. 

Anyway good work! If you have a list of upcoming features, I would be happy to contribute.

**Comment by fehmicansaglam** [comment]
Changelog.md is a good idea.

I have a couple of things in my mind. I will try to file issues about them.

Other than that if you have any other ideas it will be great pleasure for me to accept PRs.

**Comment by fehmicansaglam** [comment]
I have released 0.10.0.3
